csp-mixin/panels/azurefrontdoor.libsonnet

```diff
@@ -0,0 +1,300 @@
+              },
+              {
+                id: 'unit',
+                value: 'percent',
+              },
+            ],
+          },
+          {
+            matcher: {
+              id: 'byName',
+              options: '5xx',
+            },
+            properties: [
+              {
+                id: 'color',
+                value: {
+                  fixedColor: 'red',
+                  mode: 'fixed',
+                },
+              },
+              {
+                id: 'unit',
+                value: 'percent',
+              },
+            ],
+          },
+        ]
+      ),
+
+    afd_requests_responses_size:
+      commonlib.panels.generic.timeSeries.base.new('Request/Responses size', targets=[])
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.panelOptions.withDescription('The number of bytes sent as requests/responses from clients to AFDX and from HTTP/S proxy to clients.')
+      + this.signals.azurefrontdoorOverview.requestsSize.asPanelMixin()
+      + this.signals.azurefrontdoorOverview.responsesSize.asPanelMixin()
+      + g.panel.timeSeries.standardOptions.withUnit('decbytes')
+      + g.panel.timeSeries.fieldConfig.defaults.custom.withAxisLabel('requests(-) | responses(+)')
+      + g.panel.table.standardOptions.withOverrides(
+        [
+          {
+            matcher: {
+              id: 'byName',
+              options: 'Requests',
+            },
+            properties: [
+              {
+                id: 'custom.transform',
+                value: 'negative-Y',
+              },
+              {
+                id: 'unit',
+                value: 'decbytes',
+              },
+            ],
+          },
+        ]
+      ),
+
+    afd_total_latency:
+      this.signals.azurefrontdoorOverview.totalLatency.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.options.legend.withShowLegend(false),
+
+    afd_origin_health:
+      this.signals.azurefrontdoorOverview.originHealthPercentage.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.options.legend.withShowLegend(false)
+      + g.panel.table.standardOptions.withOverrides(
+        [
+          {
+            matcher: {
+              id: 'byName',
+              options: 'Value',
+            },
+            properties: [
+              {
+                id: 'unit',
+                value: 'percent',
+              },
+              {
+                id: 'thresholds',
+                value: {
+                  mode: 'percentage',
+                  steps: [
+                    {
+                      color: 'yellow',
+                      value: null,
+                    },
+                    {
+                      value: 30,
+                      color: 'green',
+                    },
+                    {
+                      color: 'red',
+                      value: 85,
```

Comment:
Looks like the meaning of this percent is the number of `successful health probes from AFDX to backends.` Sounds to me that the closer it gets to 100 it should be green, so:
null - red
85 - yellow
90 - green

Reply:
THis is a percentage of successful health probes. I think 0 is no probes executed